homeassistant/components/tuya/climate.py

```diff
@@ -45,6 +45,8 @@
     "wind": HVAC_MODE_FAN_ONLY,
 }
 
+PRODUCT_ID_FOR_STEP_SCALING = ['IAYz2WK1th0cMLmL']
```

Comment:
This is causing manufacturer-specific exceptions, which we do not allow.

Reply:
my discussion with an it.tuya engineer based on the results of issue #61876 and here is his suggestion.
![image](https://user-images.githubusercontent.com/18505406/153900010-ffcd355d-1f17-4f48-8ff9-95be02ce9dd4.png)